docs/about.md

```diff
@@ -0,0 +1,55 @@
+# About
+
+---
+
+## Table of Contents
+
+ - [Acknowledgments](#acknowledgments)
+ - [Contributing](#contributing)
+ - [Change Log](#change-log)
+ - [License](#license)
+
+## Acknowledgments
+
+Place for optional acknowledgments to contributors.
+
+---
+
+## Contributing
+
+Place for optional information about contributors or direction for how to contribute. For the latter --> the cadCAD [contributing](https://github.com/CADLabs/cadCAD/blob/master/CONTRIBUTING.md)  directions are useful.
```

Comment:
I agree with all these placeholders, let's fill in some MVP versions of each of these. For acknowledgements, let's follow the Ethereum Economic Model as a blueprint about how to automate this.